resources/testdata/glyphs3/NestedComponent.glyphs

```diff
@@ -39,6 +45,15 @@ name = Regular;
 );
 glyphs = (
 {
+glyphname = .notdef;
```

Comment:
I also added a .notdef glyph to aid comparisons with fontmake, since ufo2ft always adds a .notdef as the first glyph if it is not already there (we should do the same! #149)

Reply:
Good call, ty